pypush/apns/protocol.py

```diff
@@ -87,12 +87,7 @@ class FilterCommand(Command):
 
     def _lookup_hashes(self, hashes: Optional[list[bytes]]):
```

Comment:
Should a mechanism be provided to update the `KNOWN_TOPICS_LOOKUP` with additional hashes? Cache them globally somewhere?